shared/services/config/mev-boost-config.go

```diff
@@ -100,6 +100,20 @@ func NewMevBoostConfig(cfg *RocketPoolConfig) *MevBoostConfig {
 		relayMap[relay.ID] = relay
 	}
 
+	rpcPortModes := []config.ParameterOption{{
+		Name:        "Closed",
+		Description: "Do not allow connections to the RPC port",
+		Value:       config.RPC_Closed,
+	}, {
+		Name:        "Open to Localhost",
+		Description: "Allow connections from this host only",
+		Value:       config.RPC_OpenLocalhost,
+	}, {
+		Name:        "Open to External hosts",
+		Description: "Allow connections from external hosts. This is safe if you're running your node on your local network. If you're a VPS user, this would expose your node to the internet",
+		Value:       config.RPC_OpenExternal,
+	}}
+
```

Comment:
Can this also be a function call, or is it special?

Reply:
Nothing special, just forgot about this one.